smallboard/templates/base.html

```diff
@@ -53,7 +53,12 @@
 </nav>
 
 {% for message in messages %}
-<div class="alert alert-danger" style="white-space: pre-line" role="alert">{{ message }}</div>
+{% if message.level == DEFAULT_MESSAGE_LEVELS.ERROR %}
+    <div class="alert alert-danger" style="white-space: pre-line" role="alert">{{ message }}</div>
+{% endif %}
+{% if message.level == DEFAULT_MESSAGE_LEVELS.WARNING %}
+    <div class="alert alert-warning" style="white-space: pre-line" role="alert">{{ message }}</div>
```

Comment:
Can you paste a screenshot of what this looks like?

Reply:
![Screen Shot 2019-12-01 at 11 57 08 PM](https://user-images.githubusercontent.com/1312469/69931796-907d4480-1496-11ea-9d81-3621c2c41901.png)

Reply:
![Screen Shot 2019-12-02 at 12 00 03 AM](https://user-images.githubusercontent.com/1312469/69931836-b60a4e00-1496-11ea-9c55-72425dec384b.png)